### ISSUE TITLE
The way of transforming apollo color to train_id

### ISSUE THREAD

**Issue by cfzd** [cardwing/Codes-for-IntRA-KD#8]
@cardwing 
Hi, I noted that you used the equation `B x 5 + G x 3 + R` to encode the `color` to some `ids`, then map the `ids` to the `train_id` like [here](https://github.com/cardwing/Codes-for-IntRA-KD/blob/8fda71088451b81e9614974ad14fd0b05cee3878/utils/transforms_train.py#L58) and [here](https://github.com/cardwing/Codes-for-IntRA-KD/blob/8fda71088451b81e9614974ad14fd0b05cee3878/utils/transforms_train.py#L37). It's a very smart way to convert the `color` label to `train_id`.

I wonder how you figure this out or how the coefficient is obtained? The combination of 5,3 and 1 is just a guess?

**Comment by cardwing** [comment]
Yes. The coefficient is selected to make sure that each transformed id is unique.
